banners/pad/banner_var.ts

```diff
@@ -0,0 +1,73 @@
+import { createVueApp } from '@src/createVueApp';
+
+import './styles/styles.scss';
+
+import BannerConductor from '@src/components/BannerConductor/BannerConductor.vue';
+import Banner from './components/BannerCtrl.vue';
```

Comment:
1. I would request to use 'BannerCtrl' instead of 'Banner'. So that IDE can auto import it.
2. Here, we need to import 'BannerVar.vue', right?

Reply:
We name them `Banner` here because the ctrl/var files often get copied and it's easier to just change the import rather than do a find and replace. Technically we don't really care if it's the ctrl or var banner after the import.